### PR TITLE
refactor: pages

### LINKED ISSUE #1360: [REFACTOR] adjust examples to new design
**Describe the bug**
Examples need to be adjusted to new design

### PR DESCRIPTION
**Pull request by justyna-13**
# Related issue

Closes #1360 

# Scope of work

# Screenshots of visual changes

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/examples/pages/detailed-cart/_internal/OrderSummary.vue [hunk 1]
```diff
@@ -34,8 +34,13 @@
         @click="$emit('click:next')"
```
Comment (AdamPawlinski) [inline packages/vue/src/examples/pages/detailed-cart/_internal/OrderSummary.vue:34]: There are different text on buttons instead "Place my order" should be "Confirm and pay" like on design. The same on other checkout steps.

File: packages/vue/src/examples/pages/home/Home.vue [hunk 2]
```diff
@@ -6,6 +6,7 @@
         :key="index"
```
Comment (AdamPawlinski) [inline packages/vue/src/examples/pages/home/Home.vue:6]: There is no "popular categories" section and the sequence of sections is different on mobile. And there is typo in "match with it" sentence.

**Review by AdamPawlinski — CHANGES_REQUESTED** [review]
Once again, it is just amazing work @justyna-13 ! :+1:  
Only few differences with design and  I wonder if they should be necessary applied, but I have noticed them and left comments.

**Comment by AdamPawlinski** [comment]
Customization docs updated.

**Review by AdamPawlinski — DISMISSED** [review]

**Review by filrak — APPROVED** [review]

**Comment by mayashavin** [comment]
@justyna-13 please review the conflict before merging.

**Comment by justyna-13** [comment]
@mayashavin done :slightly_smiling_face:

**Pull request merged**